styles/styles.css

```diff
@@ -10,76 +11,310 @@ body {
 }
 
 .container {
-  max-width: 1440px;
-  margin-bottom: 120px;
-  padding: 120px 104px 122px;
+  max-width: 1264px;
+  padding: 0 16px;
+  margin: 0 auto;
+}
+
+input {
+  border: none;
+}
+
+.block-one__background {
+  background-image: url("../images/castelmezzano.jpg");
+  background-repeat: no-repeat;
+  background-position: top;
+  background-size: cover;
 }
 
-.title-block {
+.block-one__header {
+  padding-top: 51px;
+  padding-bottom: 202px;
   display: flex;
-  flex-direction: column;
-  justify-content: center;
+  justify-content: space-between;
 }
 
-.title {
-  text-align: center;
-  color: #383838;
-  font-size: 40px;
-  font-weight: 500;
+.block-one__header--left--svg {
+  width: 205px;
+  height: 40px;
+}
+
+.block-one__header--right {
+  display: flex;
+}
+
+.href-block {
+  margin-top: 6px;
 }
 
-.line {
+.block-one__header--right--href {
+  margin-right: 24px;
+  font-size: 24px;
+  line-height: 28px;
+  justify-content: space-around;
+  color: #ffffff;
+  text-decoration: none;
+  position: relative;
+}
+
+.block-one__header--right--href:hover::after {
+  display: block;
+  content: "";
+  height: 2px;
+  width: 91%;
   background-color: #f5bd41;
-  border: 1px solid #f5bd41;
-  width: 100px;
-  height: 1px;
-  align-self: center;
-  margin-top: 20px;
+  position: absolute;
+  left: 9%;
+  margin-top: 12px;
+}
+
+.block-one__header--right--logotip {
+  margin-left: 24px;
+  justify-content: space-around;
+  gap: 24px;
+}
+
+.block-one__header--right--logotip--night {
+  width: 30px;
+  height: 30px;
+  margin-right: 24px;
+  margin-bottom: 5px;
+}
+
+.block-one__header--right--logotip--account {
+  width: 40px;
+  height: 40px;
+}
+
+.block-one__page {
+  padding-bottom: 120px;
 }
 
-.row-quality {
+.block-one__page--header {
+  color: #ffffff;
+  font-weight: 500;
+  font-size: 50px;
+  margin-bottom: 100px;
+}
+
+.block-one__page--underform {
+  padding-left: 24px;
   display: flex;
-  margin-top: 80px;
+  gap: 221px;
```

Comment:
Bad position. Make the labels depend on their inputs.

Reply:
ok. corrected.